src/java.desktop/windows/classes/sun/awt/windows/ThemeReader.java

```diff
@@ -132,9 +122,10 @@ private static Long getTheme(String widget, int dpi) {
                 if (!valid) {
                     // Close old themes.
                     if (!dpiAwareWidgetToTheme.isEmpty()) {
-                        for (Long value :
-                             dpiAwareWidgetToTheme.get(dpi).values()) {
-                            closeTheme(value);
+                        for (Map <String, Long> dpiVal: dpiAwareWidgetToTheme.values()) {
```

Comment:
```suggestion
                        for (Map<String, Long> dpiVal : dpiAwareWidgetToTheme.values()) {
```
No space between type and its type arguments; spaces around `:` in for-each loop.